R/absolute_path_linter.R

```diff
@@ -0,0 +1,39 @@
+#' Absolute path linter
+#'
+#' Check that no absolute paths are used (e.g. "/var", "C:\\System", "~/docs").
+#'
+#' @param lax Less stringent linting, leading to fewer false positives.
+#' If `TRUE`, only lint path strings, which
+#'
+#' * contain at least two path elements, with one having at least two characters and
+#' * contain only alphanumeric chars (including UTF-8), spaces, and win32-allowed punctuation
+#'
+#' @include utils_path_linters.R
```

Comment:
This shouldn't be necessary since there is no build-time dependency on the file?

Reply:
Ah, yes, you are right. I thought we were taking the same approach here as `make_linter_from_regex()`, but we aren't.

Will change that.